lib/keycloak-admin/client/group_client.rb

```diff
@@ -35,6 +35,14 @@ def create_subgroup!(parent_id, name)
       end
       created_id(response)
     end
+    
+    def members(group_id, first=0, max=100)
+      url = "#{groups_url(group_id)}/members?first=#{first}&max=#{max}"
```

Comment:
Thanks for your contribution. 
Could you make sure the `first` and `max` parameters are integer (with `try(:to_i)`, for example). Could you also make sure the nullability values of these parameters are handled?